src/yunohost/dyndns.py

```diff
@@ -205,41 +205,45 @@ def dyndns_update(dyn_host="dyndns.yunohost.org", domain=None, key=None,
         lines = [
             'server %s' % dyn_host,
             'zone %s' % host,
-            'update delete %s. A' % domain,
-            'update delete %s. AAAA' % domain,
-            'update delete %s. MX' % domain,
-            'update delete %s. TXT' % domain,
-            'update delete pubsub.%s. A' % domain,
-            'update delete pubsub.%s. AAAA' % domain,
-            'update delete muc.%s. A' % domain,
-            'update delete muc.%s. AAAA' % domain,
-            'update delete vjud.%s. A' % domain,
-            'update delete vjud.%s. AAAA' % domain,
-            'update delete _xmpp-client._tcp.%s. SRV' % domain,
-            'update delete _xmpp-server._tcp.%s. SRV' % domain,
-            'update add %s. 1800 A %s' % (domain, ipv4),
-            'update add %s. 14400 MX 5 %s.' % (domain, domain),
-            'update add %s. 14400 TXT "v=spf1 a mx -all"' % domain,
-            'update add pubsub.%s. 1800 A %s' % (domain, ipv4),
-            'update add muc.%s. 1800 A %s' % (domain, ipv4),
-            'update add vjud.%s. 1800 A %s' % (domain, ipv4),
-            'update add _xmpp-client._tcp.%s. 14400 SRV 0 5 5222 %s.' % (domain, domain),
-            'update add _xmpp-server._tcp.%s. 14400 SRV 0 5 5269 %s.' % (domain, domain)
         ]
-        if ipv6 is not None:
-            lines += [
-                'update add %s. 1800 AAAA %s' % (domain, ipv6),
-                'update add pubsub.%s. 1800 AAAA %s' % (domain, ipv6),
-                'update add muc.%s. 1800 AAAA %s' % (domain, ipv6),
-                'update add vjud.%s. 1800 AAAA %s' % (domain, ipv6),
-            ]
+
+        dns_conf = _build_dns_conf(domain)
+        all_records = []
+        for records in dns_conf.values():
```

Comment:
Should comment on what this part of the code is doing because people won't have in mind what is coming from `_build_dns_conf` and the result of this loop.